app/webpack/stylesheets/application.scss

```diff
@@ -10,6 +10,7 @@ $govuk-font-url-function: frontend-font-url;
 $govuk-image-url-function: frontend-image-url;
 @import "~govuk-frontend/govuk/all";
 @import "core/typography";
+@import "@ministryofjustice/frontend/moj/all";
```

Comment:
Import moj frontend before typography.
The import order is important should we wish to override styles coming from either govuk or moj design systems.